mne/coreg.py

```diff
@@ -1455,11 +1458,14 @@ def _setup_fiducials(self, fids):
         # does not seem to happen by itself ... so hard code it:
         self._reset_fiducials()
 
-    def _reset_fiducials(self):  # noqa: D102
-        if self._fid_points is not None:
-            self._lpa = self._fid_points[0:1]
-            self._nasion = self._fid_points[1:2]
-            self._rpa = self._fid_points[2:3]
+    def _reset_fiducials(self):
+        dig_montage = make_dig_montage(
+            lpa=self._fid_points[0:1].squeeze(),
+            nasion=self._fid_points[1:2].squeeze(),
+            rpa=self._fid_points[2:3].squeeze(),
```

Comment:
why the squeeze? squeeze is something I want to avoid as it has a behavior that depends on input array. How about

```suggestion
            lpa=self._fid_points[0],
            nasion=self._fid_points[1],
            rpa=self._fid_points[2],
```